Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Central Repository
  *
- * Copyright 2015-2020 Basis Technology Corp.
+ * Copyright 2017-2020 Basis Technology Corp.
```

Comment:
Why does the Copyright start date change here?

Reply:
The copyright notice was copied and pasted into this file without regard for the actual date in the notice. The initial work on the feature we now know as the central repository was done in 2017. This change was part of my ongoing campaign to make the copyright notices more accurate.

Reply:
The copyright notice was copied and pasted into this file without regard for the actual date in the notice. The initial work on the feature we now know as the central repository was done in 2017. This change was part of my ongoing campaign to make the copyright notices more accurate.